Tella/Scenes/Nextcloud/ViewModels/NextcloudDraftViewModel.swift

```diff
@@ -48,6 +36,13 @@ class NextcloudDraftViewModel: DraftMainViewModel {
         }
         
         self.objectWillChange.send()
+        
+        DispatchQueue.main.async {
+            self.isValidTitle =  self.title.textValidator()
+            self.isValidDescription = self.description.textValidator()
+            self.objectWillChange.send()
+        }
+
     }
```

Comment:
Maybe we can create a function in `DraftMainViewModel` and then call it in the subclasses : 
Here an example : 
create this method inside `DraftMainViewModel` 
   ```
 func validateTitleAndDescription() {
        self.isValidTitle =  self.title.textValidator()
        self.isValidDescription = self.description.textValidator()
        self.objectWillChange.send()
    }

```
and call it here :


` self.validateTitleAndDescription()
`

The same for the other connections